File: questions/832/python/Solution.py
class Solution(object):
    def flipAndInvertImage(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        for item in A:
            start = 0;
            end = len(item) - 1;
            while (start <=end):
                temp = item[start];
                item[start]= 1 if item[end]==0 else 0
                item[end]= 1 if temp ==0 else 0
                start +=1
                end -=1
        return A

File: questions/832/java/Solution.java
class Solution {
    public int[][] flipAndInvertImage(int[][] A) {
        for(int[] item : A){
            int start = 0;
            int end = item.length -1;
            while(start <= end){
                int temp;
                temp = item[start];
                item[start] = item[end]==0 ? 1: 0;
                item[end] = temp ==0? 1 :0;
                start++;
                end--;
            }
        }
        return A;
    }
}